.github/workflows/run_tests.yml

```diff
@@ -9,12 +9,14 @@ on:
       - website/**
       - .github/**
       - "**.md"
+      - "Taskfile.yml"
   pull_request:
     branches: [ 'develop' ]
     paths-ignore:
       - website/**
       - .github/**
       - "**.md"
+      - "Taskfile.yml"
```

Comment:
this workflow ran against my PR and i certainly wouldnt expect it to. i we should exclude changes to the root Taskfile.yml from running this workflow.